flake.nix

```diff
@@ -1,42 +1,189 @@
 {
   inputs = {
-    nixpkgs.follows = "std/nixpkgs";
-    std.url = github:divnix/std;
-    tullia = {
-      url = github:input-output-hk/tullia;
-      inputs.std.follows = "std";
-    };
+    haskellNix.url = "github:input-output-hk/haskell.nix";
+    nixpkgs.follows = "haskellNix/nixpkgs-unstable";
+    iohkNix.url = "github:input-output-hk/iohk-nix";
+    flake-utils.url = "github:hamishmack/flake-utils/hkm/nested-hydraJobs";
+
+    CHaP.url = "github:input-output-hk/cardano-haskell-packages?ref=repo";
+    CHaP.flake = false;
+
+    # ciciero
+    tullia.url = "github:input-output-hk/tullia";
   };
 
-  outputs = {
-    self,
-    std,
-    tullia,
-    ...
-  } @ inputs:
-    std.growOn {
-      inherit inputs;
-      cellsFrom = nix/cells;
-      cellBlocks = [
-        (std.functions "library")
-        (std.functions "hydraJobs")
-        (tullia.tasks "pipelines")
-        (std.functions "actions")
-      ];
-    }
-    (
-      tullia.fromStd {
-        actions = std.harvest self ["cloud" "actions"];
-        tasks = std.harvest self ["automation" "pipelines"];
+  outputs = inputs:
+    let
+      profiling = false;
+      supportedSystems = [
+        "x86_64-linux"
+        "x86_64-darwin"
+        "aarch64-linux"
+        "aarch64-darwin"
+       ]; in
+    inputs.flake-utils.lib.eachSystem supportedSystems (system:
+      let
+        # setup our nixpkgs with the haskell.nix overlays, and the iohk-nix
+        # overlays...
+        pkgs = import inputs.nixpkgs {
+          overlays = [inputs.haskellNix.overlay] ++ builtins.attrValues inputs.iohkNix.overlays;
+          inherit system;
+          inherit (inputs.haskellNix) config;
+        };
+        nixpkgs = pkgs;
```

Comment:
why two names? I see you only reference nixpkgs at line 91 but that could use `pkgs` instead, or I am missing something?

Reply:
can't use pkgs there :-/ It's shadowed by another pkgs.